hack/run-test.sh

```diff
@@ -79,7 +83,8 @@ main() {
     MESSAGE="${HEADER_MESSAGE}: ${GINKGO_SUITS}"
     print ${MESSAGE}
 
-    GINKGO_FLAGS="${NO_COLOR} ${EXTRA_PARAMS} --require-suite ${GINKGO_SUITS}"
+    GINKGO_FLAGS="${NO_COLOR} ${EXTRA_PARAMS} --require-suite ${GINKGO_SUITS} ${REPORT_PARAMS}"
+    print "Command to run: GOFLAGS=-mod=vendor ginkgo ${GINKGO_FLAGS}"
```

Comment:
maybe add a `dry-run` flag, so the script just prints the full command and does not actually execute it?

Reply:
The main target here was to allow a later inspect of the params used to run ginkgo in case of test failure.
`--dry-run` will make ginkgo to walk the test hierarchy and print some additional output, even with the `succinct` flag.
That is more info that I was looking for, but if it could be useful for later debugging we can go for it.

Reply:
@ffromani makes sense? or do you still think it would be better to go for `--dry-run` option?

Reply:
Sorry, I expressed myself poorly. I meant a `--dry-run` option managed by the `hack/run-test.sh` wrapper, which then will emit (but not run) the ginkgo commands

Reply:
Added new options to `run-tests.sh` one to execute an actual `dry-run` to be able to see the list of test that would be run and another one to just see the command line without executing it.